test/NuGet.Services.Staging.Test.EndToEnd/StagingClient/IStagingClient.cs

```diff
@@ -23,7 +23,9 @@ interface IStagingClient
 
         Task<JObject> Index(string stageId);
 
-        Task<JObject> Query(string stageId, string queryString);
+        Task<JObject> Query(string stageId, string query, bool includePrerelease, int skip, int take);
+
+        Task<JObject> Autocomplete(string stageId, string q, string id, bool includePrerelease, int skip, int take);

```

Comment:
done

---
In reply to: [65844104](https://github.com/NuGet/NuGet.Staging/pull/59#discussion_r65844104) [](ancestors = 65844104)